mlflow/xgboost.py

```diff
@@ -340,6 +365,8 @@ def log_feature_importance_plot(features, importance, importance_type):
                 plt.close(fig)
                 shutil.rmtree(tmpdir)
 
+        import xgboost as xgb
```

Comment:
Looks like we already import xgboost at the top of the `autolog()` function. Do we need to import it again?